### ISSUE TITLE
scrollToOffset doesn't work when the list is scrolling

### ISSUE THREAD

**Issue by nikit-psl** [Shopify/flash-list#742]
## Current behavior

When scrolling the list if I click on an item outside the list that is supposed to take the list to the top using scrollToOffset or scrollToIndex methods, nothing happens.

https://user-images.githubusercontent.com/100762274/214251724-96fa70ff-03cf-4e8e-8baf-69bc62f8f0c7.mp4

## Expected behavior

The list should stop scrolling and move to the specified index or offset.

## To Reproduce

[Snack Repo](https://snack.expo.dev/BReqIuMDd)

## Platform:

- [ ] iOS // NOT TESTED
- [ x ] Android

## Environment

1.4.0

**Comment by RalissonMattias** [comment]
Yes, that would be a lot of help. I also have some similar cases. Properties like `onScrollToIndexFailed` would help a lot

**Comment by Nikit-Singh** [comment]
I've tried it with animated as false too, no luck.
Actually this seems to be a react native issue. This only happens in Android and I can replicate this in RecyclerListView too.

**Comment by chelovekdrakon** [comment]
Yes, instead of scroll it just "glitches"
It's only possible, when the scroll stops

**Comment by NewCoder2023** [comment]
> Can you try scrolling with `scrollToOffset({animated: false})`?

seems to work but unfortunately it glitches like "fmorau" said

**Comment by naqvitalha** [comment]
Hi! 👋

We're now focusing exclusively on **FlashList v2** and **React Native's New Architecture**. This means we're closing older issues and suggestions that aren't relevant to v2.

**If this issue/suggestion is still relevant:**
- Please verify it still occurs in FlashList v2
- Open a new issue with v2-specific details
- We'll be happy to take a look!

**Resources:**
- [Migration to v2](https://shopify.github.io/flash-list/docs/migration-guide)
- [FlashList Documentation](https://shopify.github.io/flash-list/)

Thank you for your understanding! 🙏
